packages/loader/container-loader/src/connectionStateHandler.ts

```diff
@@ -56,7 +56,7 @@ export function createConnectionStateHandler(
 ) {
     const mc = loggerToMonitoringContext(inputs.logger);
     return createConnectionStateHandlerCore(
-        mc.config.getBoolean("Fluid.Container.CatchUpBeforeDeclaringConnected") === true, // connectedRaisedWhenCaughtUp
+        mc.config.getBoolean("Fluid.Container.DisableCatchUpBeforeDeclaringConnected") !== true, // connectedRaisedWhenCaughtUp
```

Comment:
We talked briefly long ago about trying this out in the stress tests and doing some measurements.  Is that on your mind?